ENVI_ImageBandDifference.md

```diff
@@ -50,43 +52,43 @@ Mandatory (optional) settings are listed as Required = True (Required = False).
   Name  |  Required  |  Default  |  Valid Values  |  Description  
 --------|:----------:|-----------|----------------|---------------
 file_types|False|None| .hdr,.tif|GBDX Option. Comma separated list of permitted file type extensions. Use this to filter input files -- Value Type: STRING
-input_raster1|True|None| Requires two rasters to detect change, (input_raster1, input_raster2)|Specify a single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
-input_raster1_metadata|False|None| [More on ENVI input port](insert links here)|Provide a dictionary of attributes for overriding the raster metadata. -- Value Type: DICTIONARY
-input_raster1_band_grouping|False|None| [More on ENVI input port](insert links here)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
-input_raster1_filename|False|None| Requires two rasters to detect change, (input_raster1, input_raster2)|Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
-input_raster2|True|None| |Specify a second single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
-input_raster2_metadata|False|None| [More on ENVI input port](insert links here)|Provide a dictionary of attributes for overriding the raster metadata. -- Value Type: DICTIONARY
-input_raster2_band_grouping|False|None| [More on ENVI input port](insert links here)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
-input_raster2_filename|False|None| |Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
+input_raster1|True|None| A valid S3 folder containing image files. |Specify a single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
```

Comment:
@creeder23 I felt your description was implying that input_raster1 required 2 rasters. Valid value is just an S3 path??